ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ipam_host.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Infoblox, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: b1_ipam_host
+author: "Amit Mishra (@amishra), Sriram Kannan(@kannans)"
```

Comment:
Please add ur name

Reply:
Should i need to remove both names or only Amit name.